docs/reference/release-notes/highlights.asciidoc

```diff
@@ -25,6 +25,19 @@ Other versions:
 
 // tag::notable-highlights[]
 [discrete]
+[[indexing-speed-improvement]]
+===  Indexing speed improvement
+
+{es} 7.10 improves the indexing speed by 5-20%, depending on your workload.
+We've reduced the amount of coordination needed to add entries to the
+<<index-modules-translog,transaction log>>, enabling more concurrency, and
+incresed the transaction log buffer size from `8KB` to `1MB`.
+
+The performance gains are lower in intensive analysis scenarios (ie. full-text
+search analysis).
```

Comment:
I made some minor changes to improve the flow. However, the current text is
fine as-is as long as the `incresed` typo is fixed.
```suggestion
{es} 7.10 improves indexing speed by 5-20%. We've reduced the coordination
needed to add entries to the <<index-modules-translog,transaction log>>. This
reduction allows for more concurrency and increases the transaction log buffer
size from `8KB` to `1MB`.

These performance gains are lower for full-text search and other intensive
analysis scenarios.
```

Reply:
+1 to James' suggestion. I'd mention ingest pipelines too. Something along the lines of "The heavier the indexing chain, the lower the gains, so indexing chains that involve many fields, ingest pipelines or full-text indexing will see lower gains."